ctx.go

```diff
@@ -365,6 +365,41 @@ func (c *netHttpContext[B, P]) Body() (B, error) {
 	return body, err
 }
 
+// setParamValue sets a value to a reflect.Value based on its kind
+func setParamValue(value reflect.Value, paramValue string, kind reflect.Kind) error {
+	switch kind {
+	case reflect.String:
+		value.SetString(paramValue)
+	case reflect.Int, reflect.Int8, reflect.Int16, reflect.Int32, reflect.Int64:
+		intValue, err := strconv.Atoi(paramValue)
```

Comment:
Please use  strconv.ParseInt

You also need to play with the base here

```suggestion
		intValue, err := strconv.ParseInt(paramValue, 10, bitSize(kind))
```

See https://github.com/go-fuego/fuego/pull/501#discussion_r2057705693